dookug-engine/dookug-engine-pdfbox/src/main/java/hu/icellmobilsoft/dookug/engine/pdfbox/PdfBoxDocumentGenerator.java

```diff
@@ -80,41 +80,36 @@ public void generateToOutputStream(OutputStream outputStream, ParametersDataType
 
         PdfRendererBuilder pdfRendererBuilder = new PdfRendererBuilder();
 
-        // openhtmltopdf ez nelkul system.error szintre logol mindent https://github.com/danfickle/openhtmltopdf/wiki/Logging
+        // set openhtmltopdf logger (without this its logging to System.err) https://github.com/danfickle/openhtmltopdf/wiki/Logging
         XRLog.setLoggerImpl(new Slf4jLogger());
 
-        // HTML formatum kell hogy legyen, escape kezelessel nem foglalkozunk
+        // must be HTML format we dont care escaping
```

Comment:
Nem kötelező de lehet picit pontosabb így: "Must be HTML format, we don't deal with escape handling" vagy "we don't care about escape handling"

Reply:
javitottam next MRben